sqlalchemy_ibmi/backend.py

```diff
@@ -88,14 +106,46 @@ def create_connect_args(self, url):
         return [[";".join(connectors)], connect_args]
 
 
-class AS400Dialect_pyodbc(PyODBCConnector, DB2Dialect):
+class AS400DialectPyodbc(PyODBCConnector, DB2Dialect):
+    """ IBMi pyodbc sqlalchemy dialect """
+
+    def get_temp_table_names(self, connection, schema=None, **kw):
+        pass
+
+    def get_temp_view_names(self, connection, schema=None, **kw):
+        pass
+
+    def get_check_constraints(self, connection, table_name, schema=None, **kw):
+        pass
+
+    def get_table_comment(self, connection, table_name, schema=None, **kw):
+        pass
+
+    def do_begin_twophase(self, connection, xid):
+        pass
+
+    def do_prepare_twophase(self, connection, xid):
+        pass
+
+    def do_rollback_twophase(self, connection, xid, is_prepared=True,
+                             recover=False):
+        pass
+
+    def do_commit_twophase(self, connection, xid, is_prepared=True,
+                           recover=False):
+        pass
+
+    def do_recover_twophase(self, connection):
+        pass
+
+    def get_isolation_level(self, dbapi_conn):
+        pass
+
     supports_unicode_statements = True
     supports_sane_rowcount = False
     supports_sane_multi_rowcount = False
-    supports_native_decimal = True
```

Comment:
Looks like this was duplicated previously. The question is: which value _should_ it have? I suggest we add a TODO to investigate further. I can't find much on what this does, other than it is used by the Oracle and possibly MySQL interfaces.

Certainly, Db2 has a native DECIMAL type _and_ the driver handles it, but this setting seems to be about something a little different so I'm not sure.